mautic/mautic
# Upgrade ckeditor 4 to 5

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | Yes
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | No 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Upgraded ckeditor 4 to 5 as [version 4 is end of life](https://ckeditor.com/blog/ckeditor-4-end-of-life/).



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. run `npm install` (for generate ckedior.js file) and `mautic:assets:generate` command  (for generate other js/css files) 
3. Test complete application where ckeditor is added (anywhere you can add rich text)
4. Be sure to test the GrapesJS builder in emails and landing pages.
